ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -163,13 +163,19 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
         // Format is determined by series_name_format in query spec
         const formattedSeriesName = timeSeries.formattedName ?? timeSeries.name;
 
-        const seriesColor = getSeriesColor(
-          formattedSeriesName,
-          seriesIndex,
-          echartsPalette as string[],
-          visual.palette?.kind
-        );
-        seriesIndex++; // Used for repeating colors in Categorical palette
+        // Fallback is unlikely to set unless echarts theme palette in charts theme provider is undefined.
```

Comment:
Do you mean that the fallback is unlikely to be used unless `echartsPalette` is undefined?

I thought RedHat was the only one that was using the eCharts palette?

Reply:
Just talked with Mohit and we're going to change the hex values to be more colorblind friendly, I'll make this change as part of that PR!